framework/src/wifi_manager/wifi_manager.c

```diff
@@ -30,6 +30,13 @@
 #include "wifi_utils.h"
 #include "wifi_profile.h"
 
+#ifdef CONFIG_WIFIMGR_ERROR_REPORT
+#include <error_report/error_report.h>
+#define WIFIADD_ERR_LOG(reason_code)	RT_ERR_DATA_CREATE(ERRMOD_WIFI_MANAGER, reason_code)
```

Comment:
I can't know what is differece between it and dbg by API name,
because XXX_ERR_LOG looks like printing err message thru serial. 

How about "WIFI_ERR_REPORT``` or else?

Reply:
WIFIADD_ERR_LOG is different from dbg messages in that it will make a call to the error reporting API via the RT_ERR_DATA_CREATE macro. WIFI_ERR_REPORT is too vague, as a report is a collection of several logs, and in this case, we are creating one log.

Reply:
Modified as per suggestions. Thank you.